### DISCUSSION TITLE
Trying to add insecure registry

### DISCUSSION THREAD

**Discussion by TheGreatBey0nd** [rancher-sandbox/rancher-desktop#870] [General]
Hi,
i  run rancher desktop on my windows machine and wsl2. Is it possible to add insecure registries? 

I have trying to create docker credentials (https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#registry-secret-existing-credentials), editing the docker config file (~/.docker/config.json) and try to editing the containerd config file (/etc/containerd/config - haven´t found it in wls2).

On the commandline "nerdctl --insecure-registry pull <img>" image pulling works fine.

**Comment by w7089** [comment]
@TheGreatBey0nd did you succeed to add insecure registry to rancher desktop docker daemon config?

**Reply by TheGreatBey0nd** [reply]
No, but we have found a way to make our Nexus a secure registry.

**Reply by chipmanc** [reply]
How?

**Comment by xi2656-akhanna** [comment]
I was able to add a private registry by running
`wsl -d rancher-desktop -e vi /etc/docker/daemon.json`

and add following lines
 ```
{  
    "insecure-registries": [  
        "url-registry-url"  
    ]  
}  
```

Restart Rancher. But this is a temporary solution and has to be done every time your machine starts
